cmd/antrea-agent/agent.go

```diff
@@ -126,13 +130,13 @@ func run(o *Options) error {
 		TransportIface:        o.config.TransportInterface,
 	}
 
-	wireguardConfig := &config.WireGuardConfig{
-		Port: o.config.WireGuard.Port,
-	}
-	routeClient, err := route.NewClient(serviceCIDRNet, networkConfig, o.config.NoSNAT)
+	routeClient, err := route.NewClient(serviceCIDRNet, networkConfig, o.config.NoSNAT, o.config.AntreaProxy.ProxyAll)
 	if err != nil {
 		return fmt.Errorf("error creating route client: %v", err)
 	}
+	wireguardConfig := &config.WireGuardConfig{
```

Comment:
this order change seems pointless?

Reply:
Sorry, I didn't notice this when rebasing.